### PR TITLE
Add admin

### PR DESCRIPTION
**Pull request by TheEyePatch**
This PR is mainly for generating the controller, views, and RSpec test for admin. The admin is created by inheriting from the User model. The registration is skipped in the admin devise, the first admin account is created using the console, while the following admin accounts are created when an admin account is logged in.
The admin is able to see the list of all users that are approved and the agencies that are still pending.

### Admin Sign In Page
admin_sign_in_travelApp

### All User Page
admin_all_users_travelApp

### Admin Profile Page
admin_profile_page_travelApp

### CODE REVIEW

File: app/views/devise/sessions/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+<% if current_page?(new_tourist_session_path || new_agency_session_path) %>
+  <%= render "devise/shared/links" %>
+<% end %>
+
```
Comment (dlanoronald123) [inline app/views/devise/sessions/new.html.erb:28]: Can you please render the design I created? Thanks mate!

**Review by dlanoronald123 — CHANGES_REQUESTED** [review]
Please submit review

**Review by dlanoronald123 — APPROVED** [review]

File: app/views/devise/sessions/new.html.erb (hunk 1 above)
Comment (TheEyePatch) [reply app/views/devise/sessions/new.html.erb:28]: I will add in the next PR

**Comment by Eileenandrea** [comment]
approve

**Review by morpheus17 — APPROVED** [review]
validated, just missing design from dummy app

**Review by Eileenandrea — APPROVED** [review]

**Pull request merged**